### PR TITLE
Update the community page

### LINKED ISSUE #15: Update community.html
Replace content in community.html with links and details for the community member persona. Apply new layout.

### PR DESCRIPTION
**Pull request by oraNod**
Fixes  #15

### CODE REVIEW

**Review by samccann — APPROVED** [review]
LGTM but might want other team members to pipe in on this one

**Comment by cybette** [comment]
I think it looks fine for docsite, however we need to rethink how it will work in the new community site. Will it be:

ansible.community
- docsite
- join the community!
- blog
- etc.

or

ansible-community
- docsite
  - join the community!
- blog
- etc.

**Comment by oraNod** [comment]
@cybette I think it will be option 1 there. This PR is just for the docsite nav. If ansible.community goes ahead we'll have a new nav bar at the top that applies to all pages, including docsite.

**Comment by cybette** [comment]
@oraNod sounds good, thanks! LGTM

File: data/links.yaml [hunk 1]
```diff
@@ -104,6 +104,22 @@ project_doc_home:
...
+  discuss:
+    email: "https://docs.ansible.com/ansible/latest/community/communication.html#asking-questions-over-email"
+    chat: "https://matrix.to/#/#social:ansible.com"
+    irc: "https://docs.ansible.com/ansible/latest/community/communication.html#general-channels"
+  join:
+    conduct: "https://docs.ansible.com/ansible/latest/community/code_of_conduct.html"
+    working_groups: "https://docs.ansible.com/ansible/latest/community/communication.html#working-groups"
+    public_meetings: "https://docs.ansible.com/ansible/latest/community/communication.html#meetings-on-chat"
+  subscribe:
+    bullhorn: "https://eepurl.com/gZmiEP"
```
Comment (cybette) [inline data/links.yaml:121]: Can we use this url: https://bit.ly/subscribe-bullhorn

**Review by cybette — APPROVED** [review]

**Review by GregSutcliffe — APPROVED** [review]
Not going to comment on style, you really don't want my input on that. Content LGTM for how things stand today, thanks!

**Comment by oraNod** [comment]
Thanks all. Going to merge this one.

**Pull request merged**